components/Profile.js

```diff
@@ -0,0 +1,30 @@
+import getImg from '../lib/prefix'
+import profile from '../styles/Profile.module.scss'
+import { FaFacebookF, FaInstagram, FaGithub } from 'react-icons/fa';
+import { MdEmail } from 'react-icons/md';
+
+export default function Profile({name, imagePath, facebook, instagram, github, email}) {
+    return (
+        <>
+            <div className={profile.profileContainer}>
+                <div className={profile.innerContainer}>
+                    <img className={profile.image} src={getImg(imagePath)}></img>
+                    
+
+                    <div className={profile.textContainer}>
+                        <h2 className={profile.text}>{name}</h2>
+                        <h3 className={profile.text}>President</h3>
+                    </div>
+                    
+                    <div className={profile.iconsContainer}>
+                        {facebook && <a target="_blank" href={facebook}><FaFacebookF/></a>}
```

Comment:
Must add `rel="noreferrer"` 

Build error:
"Error: Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener  react/jsx-no-target-blank"